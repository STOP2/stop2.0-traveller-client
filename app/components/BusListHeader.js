import React, { Component } from 'react';
import { View } from 'react-native';

import { DefaultText } from '../components/textComponents';

import styles from '../styles/stylesheet';
import strings from '../resources/translations';

const BusListHeader = () => (
          <View style={styles.busrowheader}>
            <DefaultText style={styles.busrowText}>{strings.type}</DefaultText>
            <DefaultText style={styles.busrowText}>{strings.line}</DefaultText>
            <DefaultText style={styles.busrowText2}>{strings.dest}</DefaultText>
            <DefaultText style={styles.busrowText}>{strings.leaves}</DefaultText>
          </View>
);

const RouteListHeader = () => (
          <View style={styles.busrowheader}>
            <DefaultText style={styles.busrowText}>{strings.stopCode}</DefaultText>
            <DefaultText style={styles.busrowText2}>{strings.stopName}</DefaultText>
            <DefaultText style={styles.busrowText}>{strings.arrivesIn}</DefaultText>
          </View>
);
