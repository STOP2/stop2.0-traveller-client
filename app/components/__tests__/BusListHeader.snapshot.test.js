import 'react-native'
import React from 'react'
import BusListHeader from '../BusListHeader'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'


jest.mock('react-native-localization')

describe('BusListHeader component', () =>
{
    it('renders correctly', () =>
    {
        let tree = renderer.create(<BusListHeader />).toJSON()

        expect(tree).toMatchSnapshot()
    })
})
