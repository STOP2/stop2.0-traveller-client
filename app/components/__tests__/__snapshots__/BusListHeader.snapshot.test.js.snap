exports[`BusListHeader component renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E9E9E9",
      "flexDirection": "row",
      "padding": 10,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "gotham-rounded-book",
        },
        Object {
          "flex": 1,
          "fontSize": 15,
        },
      ]
    }>
    Type
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "gotham-rounded-book",
        },
        Object {
          "flex": 1,
          "fontSize": 15,
        },
      ]
    }>
    Line
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "gotham-rounded-book",
        },
        Object {
          "flex": 3,
          "fontSize": 15,
        },
      ]
    }>
    Destination
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "gotham-rounded-book",
        },
        Object {
          "flex": 1,
          "fontSize": 15,
        },
      ]
    }>
    Leaves
  </Text>
</View>
`;
